pom.xml

```diff
@@ -102,12 +102,12 @@
 			<dependency>
 				<groupId>org.apache.logging.log4j</groupId>
 				<artifactId>log4j-core</artifactId>
-				<version>2.3</version>
+				<version>[2.13.2,)</version>
 			</dependency>
```

Comment:
I'm not familiar with this format of version syntax, but I trust you know what's going on